core/domain/subtopic_page_services_test.py

```diff
@@ -117,3 +164,12 @@ def test_commit_log_entry(self):
             subtopic_page_commit_log_entry.subtopic_page_id,
             self.subtopic_page_id)
         self.assertEqual(subtopic_page_commit_log_entry.user_id, self.user_id)
+
+    def test_delete_subtopic_page(self):
+        subtopic_page_id = (
+            subtopic_page_domain.SubtopicPage.get_subtopic_page_id(
+                self.TOPIC_ID, 1))
+        subtopic_page_services.delete_subtopic_page(
+            self.user_id, self.TOPIC_ID, 1)
+        with self.assertRaises(base_models.BaseModel.EntityNotFoundError):
```

Comment:
This is a good assertion, as it specifies what the exact exception is.

Reply:
Changed all the above.